manifests/backup.pp

```diff
@@ -23,14 +23,23 @@
 #
 # Copyright 2014 Spencer Owen, unless otherwise noted.
 #
-class gitlab::backup inherits ::gitlab {
+class gitlab::backup (
+  $backup_hour     = 2,
+  $backup_minute   = 0,
+  $backup_month    = undef,
+  $backup_monthday = undef,
+  $backup_weekday  = undef,
```

Comment:
why not just take these options from init?

or are you trying to make it explicitly possible to call this class from outside of spuder/gitlab?